WordPress/src/main/java/org/wordpress/android/ui/jetpackoverlay/JetpackFeatureRemovalOverlayUtil.kt

```diff
@@ -27,12 +27,9 @@ class JetpackFeatureRemovalOverlayUtil @Inject constructor(
     private val buildConfigWrapper: BuildConfigWrapper,
     private val dateTimeUtilsWrapper: DateTimeUtilsWrapper,
 ) {
+    @Suppress("unused", "UNUSED_PARAMETER")
     fun shouldShowFeatureSpecificJetpackOverlay(feature: JetpackOverlayConnectedFeature): Boolean {
-        return !buildConfigWrapper.isJetpackApp && isWpComSite() &&
-                isInFeatureSpecificRemovalPhase() && hasExceededOverlayFrequency(
-                feature,
-                getCurrentPhasePreference()!!
-        )
+        return true
```

Comment:
This change is for testing purposes only. This change will make sure that the overlay is shown everytime the features are accessed.